scholarshipr/index.js

```diff
@@ -364,15 +352,17 @@ function getCardHTML(scholarship) {
 	returnableString += "</li>    <li class=\"list-group-item\">";
 	returnableString += "GPA Requirement: " + gpaRequirement;
 	returnableString += "</li> </ul>  <div class=\"mx-auto\"><div class=\"card-body\">";
+	returnableString += '<td>' + 
+						` <a class="btn btn-success btn-sm mr-1" href="#" onclick="if (confirm('Approve scholarship &quot;` + scholarship.name + `&quot;?')) { window.location = '/approve?_id=` + scholarship.id + `' }">Approve</a>` +
```

Comment:
I think this should only show up if the scholarship is pending. Right now there's an "Approve" button on the detail view of even approved scholarships.

Reply:
Thanks, will fix it!